### ISSUE TITLE
Remove dependency on MVC

### ISSUE THREAD

**Issue by gimmi** [moodmosaic-attic/EmptyWebApiProjectTemplate#3]
The project reference System.Web.Mvc DLL that is parto of ASP.NET MVC 4. This cause problem when deploying on machines that doesn not have neither .NET 4.5 and MVC4 installed.

**Comment by moodmosaic** [comment]
This extension provides an empty template for the Web API projects under ASP.NET MVC 4. Referencing ASP.NET MVC 4 specific DLLs is necessary in order to have a template that can compile.

**Comment by gimmi** [comment]
I had problem deploying a pure WebAPI app built from this template in a server that has no .NET4.5 nor MVC4 installed. The error was:

```
Could not load file or assembly 'System.Web.Mvc, Version=4.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35' or one of its dependencies. The system cannot find the file specified. 
```

Looking in the project reference i found thaty `System.Web.Mvc` is piched from MVC4 installation (see screenshot). Removing the reference and the `AreaRegistration.RegisterAllAreas();` code in `Clobal.asax` resolved the problem.

IMO for a _pure_ WebAPI project that DLL is useless.

![Capture](https://f.cloud.github.com/assets/6589/110882/3b9ae502-6ada-11e2-97d8-039f8b9f6c53.PNG)

**Comment by moodmosaic** [comment]
I see the point.. I am going to look at it and, if necessary, I will release an updated version. Thank you :)

**Comment by moodmosaic** [comment]
That's actually a great idea! It is now available in version 5.0 on the Visual Studio gallery [page](http://visualstudiogallery.msdn.microsoft.com/a989a149-4bc3-4292-ac8a-5101ee1722d7).
